src/model/EnergyManagementSystemActuator.hpp

```diff
@@ -124,14 +138,16 @@ namespace model {
     friend class Model;
     friend class IdfObject;
     friend class openstudio::detail::IdfObject_Impl;
+
```

Comment:
The other EMS objects have these as private as well.  i dont remember why, but at this point its probably not a big deal?

Reply:
No problem no. As long as it works. FYI I added an in-depth waltkrhough of my changes just now. use that to navigate the changes @brianlball